internal/commands/app/create_inputs.go

```diff
@@ -101,29 +114,54 @@ func (i *createInputs) resolveDirectory(wd string) (string, error) {
 	return fullPath, nil
 }
 
-func (i *createInputs) resolveDataSource(client atlas.Client, groupID string) (dataSource, error) {
+func (i *createInputs) resolveCluster(client atlas.Client, groupID string) (clusterService, error) {
 	clusters, err := client.Clusters(groupID)
 	if err != nil {
-		return dataSource{}, err
+		return clusterService{}, err
 	}
 	var clusterName string
 	for _, cluster := range clusters {
-		if i.DataSource == cluster.Name {
+		if i.Cluster == cluster.Name {
```

Comment:
[q] Do we care about casing here? It looks like in Atlas, the UI doesn't stop you from creating a cluster with the same name if the casing is different, but the cluster doesnt actually get created (so I assume this is a bug on the former). If we don't want to make any underlying assumptions, we can enforce casing but I wonder if we should note that somewhere since imagine devs could run into this often

Reply:
I think for simplicity's sake (and for parity with the UI, broken or not) we should also just leave this check as-is.  good callout though, and something we may want to keep in mind as we get more usage with these flags